third-party/realdds/src/dds-device-server.cpp

```diff
@@ -237,114 +256,140 @@ void dds_device_server::on_control_message_received()
             continue;
 
         _control_dispatcher.invoke(
-            [x = std::move( data ), this]( dispatcher::cancellable_timer )
+            [j = data.json_data(), sample = info, this]( dispatcher::cancellable_timer )
```

Comment:
Why not moving the json_data? I don't see it is used later

Reply:
Not sure I follow, you're asking why it's not `j = std::move( data.json_data() )`?
`data.json_data()` is rvalue, the compiler should take care of the move automatically when it initializes the local member `j`.